SPEC.md

```diff
@@ -211,6 +211,65 @@ Plugins may define additional fields that they accept and may generate an error
 }
 ```
 
+### Network Configuration Lists
+
+Network configuration lists provide a mechanism to run multiple CNI plugins for a single container in a defined order, passing the result of each plugin to the next plugin.
```

Comment:
True. But if I understand correctly, what you are describing seems to be internal details of Kubernetes, in other words Kubernetes perspective of how a CNI network (here a combination of portmapping and real driver) is exposed to the user? I believe the spec should be independent of assumptions of the internal details of the container runtime. So still trying to understand your argument as why the internal specifics of Kubernetes should be influential in ratifying the spec?